python/samples/kernel-syntax-examples/openapi_example/openapi_client.py

```diff
@@ -7,14 +7,14 @@
     """Client"""
     kernel = sk.Kernel()
 
-    openapi_skill = register_openapi_skill(kernel, "openApiSkill", "openapi.yaml")
+    openapi_skill = register_openapi_skill(kernel, "openApiSkill", "samples/kernel-syntax-examples/openapi_example/openapi.yaml")
```

Comment:
The README calls out to navigate to In a terminal, navigate to `semantic_kernel/python/samples/kernel-syntax-examples/openapi_example` which is inconsistent with this change